menu.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <section class="headline">
+   <div class="works">
+      <h2>My Recent Works</h2>
+      <img src="images/icons/Indicator.svg" alt="Indicator"><br>
+    <div class="work-list">
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>Ruby On Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
+      </div> 
+        </div>
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>RubyOnRails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
+      </div> 
+        </div>
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>RubyOnRails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
+      </div> 
+        </div>
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>RubyOnRails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
+      </div> 
+        </div>
+       </div>
+      </div>
+      <div class="About">
+        <h1>About me</h1>
```

Comment:
- As I suggested in the preview review, the title `About me` is currently a h1 tag. but you are meant to have only one `h1` tag on your page and that should be the main title, I'll suggest you change this to `h2 or h3`. :smile: 
- [ ] Also your about me section is missing the two background images that are present on the Figma design, I'll suggest you get the images from the design and implement them accordingly, :+1: